src/Worker.Extensions.DurableTask/Exceptions/DurableSerializationException.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.DurableTask.Protobuf;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask.Exceptions;
+
+internal class DurableSerializationException : Exception
+{
+    private Exception FromException;
```

Comment:
Fields should be lowercased and, whenever possible, should be `readonly`.
```suggestion
    private readonly Exception fromException;
```

Reply:
Done